src/main/java/com/treasuredata/client/TDClientHttpConflictException.java

```diff
@@ -26,8 +26,22 @@
 public class TDClientHttpConflictException
         extends TDClientHttpException
 {
+    private String conflictsWith;
```

Comment:
Good catch! It should be final. But using `Optional` for fields is not recommended.